### ISSUE TITLE
MeTube Website Improvement

### ISSUE THREAD

**Issue by perkyPearl** [Open-Source-Chandigarh/MeTube#19]
Hi, I've noticed a few areas on your YouTube Video platform that could benefit from some improvements to make the experience more appealing and user-friendly. I believe that I can make the changes and make the Platform more appealing. 

Additionally, it would be ideal to implement this using React. If you'd like, I can assist in "Reactizing" the platform to achieve these improvements.

Let me know if you're interested!

**Comment by MITARPALGROVER** [comment]
yeah i can assign you this issue but the thing is i am a beginner and i dont know react so i would not be able to understand what changes you made
